src/server/svr_mail.c

```diff
@@ -255,7 +259,7 @@ svr_mailowner_id(char *jid, job *pjob, int mailpoint, int force, char *text)
 
 	/* setup sendmail command line with -f from_whom */
 
-	margs[0] = SENDMAIL_CMD;
+	margs[0] = mailer;
```

Comment:
This block of code to setup the sendmail execution seems exactly same between here and in the reservation case. This is not your code change, something older. Care to make this a common function for better modularity?

Reply:
OK, please see the suggested changes. Not sure how large block of code do you mean... I added svr_exec_mailer() function not only for setup but for also for execv. Redundancy removed.